hooks/post_gen_project.py

```diff
@@ -1,22 +1,15 @@
 import uuid
 import re
-import sys
 
 from pathlib import Path
 
-project_id = f"neuro-project-{uuid.uuid4().hex[:8]}"
+project_kebab = "{{ cookiecutter.project_slug }}".replace("_", "-")
+project_id = f"{project_kebab}-{uuid.uuid4().hex[:8]}"
 
-makefile_path = Path("Makefile")
+live_file_path = Path(".neuro/live.yml")
 
-text = makefile_path.read_text()
-text = re.sub(r"PROJECT_ID=placeholder", f"PROJECT_ID={project_id}", text)
+text = live_file_path.read_text()
+text = re.sub(r"project:project-kebab-name-placeholder", f"project:{project_kebab}", text)
```

Comment:
what's kebab? :sweat_smile:

Reply:
it a `kebab-case` 😄 (like a camelCase)
I just found out that the meaning is the same as `slug-case`, changed it back in 3524734.)